pom.xml

```diff
@@ -126,6 +126,12 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>org.glassfish.hk2</groupId>
+                <artifactId>hk2-utils</artifactId>
+                <version>2.4.0-b32</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
When I upgraded, this was missing. I believe rest-assured uses it in our integration tests.